### DISCUSSION TITLE
nodejs-poolController & Pentair Cover Cards

### DISCUSSION THREAD

**Discussion by nrobfd** [tagyoureit/nodejs-poolController#783] [Ideas]
I have an Intellicenter, IC60, and the add-on cover card. On the panel, I can set what percentage I want the IC60 to operate at when the cover is closed. But only on the panel. The app, web portal, and nodejs-poolController settings only affect what the IC60 does when the cover is open.

If it's even possible, It would be fantastic if nodejs-poolController could add functionality to give us the ability to change what the IC60 does when covered.

**Comment by rstrouse** [comment]
Please capture a replay of setting this position including what is was set to and what you set it to.  Where is the interface for this is it on the screen where you set the chlorinator to where you set the cover?

**Reply by nrobfd** [reply]
On the intellicenter main panel, you select Settings > Advanced System Configuration > Installation Setup > Pool/Spa Cover > Cover 1 and then you change change the settings. On this replay, I changed the when covered IC60 settings from 18% to 20%.

[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/11707961/replay.zip)

**Reply by rstrouse** [reply]
Please pull (update) njsPC then do the replay again.  This time do not check Capture Configuration Reload checkbox.

**Reply by nrobfd** [reply]
Ok. Should be up to date now. Here's the new capture:

[replay-2.zip](https://github.com/tagyoureit/nodejs-poolController/files/11709433/replay-2.zip)

**Reply by rstrouse** [reply]
For the replay provided what was the covered setting vs the uncovered setting value.  I assume the covered setting was 20% but what was the uncovered setting?

**Reply by nrobfd** [reply]
For replay 2, I changed the covered setting from 20 to 18. The uncovered setting is 20.

**Reply by rstrouse** [reply]
Well let me just say booh to that.  The panel is not publishing anything wrt the chlorinator during that transition.  It appears all the action is going on at the panel and it never makes it out.

**Reply by nrobfd** [reply]
Bummer. It seems like the only thing that gets out then is whether it is covered or uncovered. Thanks for exploring this! Hopefully, Pentair will increase the functionality in the future.
